pkgs/development/tools/misc/edb/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, mkDerivation, fetchFromGitHub, cmake, pkgconfig, boost, capstone
+, double-conversion, graphviz, qtxmlpatterns }:
+
+mkDerivation rec {
+  pname = "edb";
+  version = "2020-08-21";
+
+  src = fetchFromGitHub {
+    owner = "eteran";
+    repo = "edb-debugger";
+    rev = "3660d40f5beb0321138f03d3aaf7d591408dcc7c";
+    fetchSubmodules = true;
+    sha256 = "1v4rvbr26b2xjsnpw5j6037q6pyisjjmwpxqziwbiijdsi9cgf0f";
+  };
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+
+  buildInputs = [ boost.dev capstone double-conversion graphviz qtxmlpatterns ];
+
+  postPatch = ''
+    # Remove CMAKE_INSTALL_PREFIX from DEFAULT_PLUGIN_PATH otherwise the nix store path will appear twice.
+    substituteInPlace ./src/CMakeLists.txt --replace \
+        '-DDEFAULT_PLUGIN_PATH=\"''${CMAKE_INSTALL_PREFIX}/''${CMAKE_INSTALL_LIBDIR}/edb\"' \
+        '-DDEFAULT_PLUGIN_PATH=\"''${CMAKE_INSTALL_LIBDIR}/edb\"'
+
+    # The build script checks for the presence of .git to determine whether
+    # submodules were fetched and will throw an error if it's not there.
+    # Avoid using leaveDotGit in the fetchFromGitHub options as it is non-deterministic.
+    mkdir -p src/qhexview/.git
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Cross platform AArch32/x86/x86-64 debugger";
+    homepage = "https://github.com/eteran/edb-debugger";
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ lihop ];
+    platforms = platforms.linux;
```

Comment:
according to ofborg, this is not supported on aarch64

Reply:
How can this be resolved?

Reply:
How can this be resolved?

Reply:
grep suggests `platforms = ["x86_64-linux" "i686-linux"];`